### PR TITLE
Glossary crud, update CSV loader

### PR DESCRIPTION
**Pull request by afrasier**
- Add glossary create, update
- Update CSV loader
- Add tests
- Add permissions (Jenny defaults to having permission)

I didn't add "delete" functionality since I think we'd want to put history tracking on it as well if that was the case. We can create another story for that if we think we'll need that functionality.

### CODE REVIEW

**Comment by mjoyce91** [comment]
Good idea on not adding deleting. @krisdelacruz actually mentioned using archiving instead of deleting. Maybe add a flag to denote it as archived, which would cause it not to display in the public glossary?

**Comment by afrasier** [comment]
Yeah, would we want to always filter it API side, or leave it available but the front end just asks for `?is_archived=false`

**Comment by mjoyce91** [comment]
Either is fine

File: talentmap_api/user_profile/management/commands/create_seeded_users.py [hunk 1]
```diff
@@ -14,16 +14,16 @@ class Command(BaseCommand):
...
-        ("shadtrachl", "shadtrachl@state.gov", "password", "Leah", "Shadtrach", False, True),
-        ("woodwardw", "woodwardw@state.gov", "password", "Wendy", "Woodward", True, False)
+        ("guest", "guest@state.gov", "guestpassword", "Guest", "McGuestson", False, False, []),
+        ("admin", "admin@talentmap.us", "admin", "Administrator", "TalentMAP", False, False, []),
+        ("doej", "doej@talentmap.us", "password", "John", "Doe", False, False, []),
+        ("townpostj", "townpostj@state.gov", "password", "Jenny", "Townpost", False, False, ["glossary_editors"]),
+        ("batisak", "batisak@state.gov", "password", "Kara", "Batisak", False, False, []),
+        ("rehmant", "rehmant@state.gov", "password", "Tarek", "Rehman", False, False, []),
+        ("shadtrachl", "shadtrachl@state.gov", "password", "Leah", "Shadtrach", False, True, []),
+        ("woodwardw", "woodwardw@state.gov", "password", "Wendy", "Woodward", True, False, [])
```
Comment (burgwyn) [inline talentmap_api/user_profile/management/commands/create_seeded_users.py]: Did we lose Wendy's bureau_ao role?

Comment (afrasier) [reply talentmap_api/user_profile/management/commands/create_seeded_users.py]: No, it's part of the is_ao flag. I just got lazy and didn't refactor how those were set. :)

**Pull request merged**